.github/workflows/test_with_woodwork_main_branch.yml

```diff
@@ -0,0 +1,59 @@
+name: Test Featuretools with Woodwork main Branch
+on:
+  schedule:
+    - cron: '0 * * * *'
+  pull_request:
+    types: [opened, synchronize]
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+jobs:
+  scheduled_unit_latest_tests:
+    name: ${{ matrix.python_version }} unit tests ${{ matrix.libraries }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        python_version: ["3.7", "3.8", "3.9", "3.10"]
+        libraries: ["core", "koalas", ]
+    steps:
+      - name: Set up python ${{ matrix.python_version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python_version }}
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Build featuretools package
+        run: make package_featuretools
+      - name: Set up pip and graphviz
+        run: |
+          pip config --site set global.progress_bar off
+          python -m pip install -U pip
+          sudo apt update && sudo apt install -y graphviz
+
+      - if: ${{ matrix.libraries == 'core' }}
+        name: Install Woodwork & Featuretools - test requirements
+        run: |
+          python -m pip install -e unpacked_sdist/[test]
+          python -m pip install -U git+https://github.com/alteryx/woodwork
+      - if: ${{ matrix.libraries == 'dask' }}
+        name: Install Woodwork & Featuretools - dask requirements
+        run: |
+          python -m pip install -e unpacked_sdist/[test]
+          python -m pip install -U git+https://github.com/alteryx/woodwork dask[dataframe]>=2021.10.0
+      - if: ${{ matrix.libraries == 'koalas' && matrix.python_version != '3.10' }}
+        name: Install Woodwork & Featuretools with koalas pkg - koalas requirements
+        run: |
+          sudo apt install -y openjdk-11-jre-headless
+          JAVA_HOME="/usr/lib/jvm/java-11-openjdk-amd64"
+          PYSPARK_SUBMIT_ARGS="--master local[2] pyspark-shell"
+          python -m pip install -e unpacked_sdist/[koalas]
+          python -m pip install -e unpacked_sdist/[test]
+          python -m pip install -U git+https://github.com/alteryx/woodwork
```

Comment:
This command is shared with the "core" install

Assuming the exclude change works, we could make these two lines be a shared step, with the earlier lines of this step becoming a koalas-only step that occurs before the shared step

Reply:
updated